### PR TITLE
Kill the dotnet process if it's running when we try to runtime

### PR DESCRIPTION
**Pull request by jepetty**
A user hit a problem where the dotnet process was still running when we tried to install the proper runtime we need, causing the install to fail. This kills the dotnet process first so that we can successfully install the runtime.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
 		"@actions/github": "^4.0.0",
 		"@actions/io": "^1.0.2",
 		"@actions/tool-cache": "^1.3.0",
-		"@richnav/rich-code-nav-indexer-core": "^0.1.56-alpha",
+		"@richnav/rich-code-nav-indexer-core": "^0.1.61-alpha",
```
Comment (AArnott) [inline package.json:19]: I'm trying to find a match in [this history](https://dev.azure.com/devdiv/DevDiv/_git/RichCodeNavIndexingCore?_a=history) so I can see how it is that we kill the dotnet process, but I can't find any commit that matches the PR's title even roughly. Can you help?

Comment (jepetty) [reply package.json:19]: [Here](https://dev.azure.com/devdiv/DevDiv/_git/RichCodeNavIndexingCore/commit/ed7195b7b808d2281b53d2c1bc60c7637788062a?refName=refs%2Fheads%2Fmain) is the merged PR commit. In particular, line 37 has the explicit command

Comment (AArnott) [reply package.json:19]: Thanks. 
Do we know what that dotnet process is doing? If it's a build server, we can potentially shut it down more politely using `dotnet build-server shutdown`.

**Review by AArnott — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jepetty) [reply package.json:19]: Thanks! You were right, it was a build server. [Here's](https://github.com/AmadeusW/NuGetPackageExplorer/runs/1098405635?check_suite_focus=true) a build that demonstrates `dotnet build-server shutdown` resolves the issue, and [here's](https://devdiv.visualstudio.com/DevDiv/_git/RichCodeNavIndexingCore/pullrequest/273797?_a=overview) and updated PR using your suggestion. I'll wait to merge this pull request until we've incorporated the graceful shutdown.

**Pull request merged**